test/e2e/atlas/search_test.go

```diff
@@ -70,8 +70,10 @@ func TestSearch(t *testing.T) {
 	"collectionName": "{{ .collectionName }}",
 	"database": "test",
 	"name": "{{ .indexName }}",
-	"mappings": {
```

Comment:
fyi: changes in the search API I was talking about @jeroenvervaeke @tibulca @fmenezes, not sure if this file format is the one local uses or if mongot will get updated

Reply:
I think we will need to update https://www.mongodb.com/docs/atlas/atlas-search/index-definitions/ or at least https://www.mongodb.com/docs/atlas/cli/current/command/atlas-clusters-search-indexes-update/#std-label-atlas-clusters-search-indexes-update